src/components/utils/test/thread_test.cc

```diff
@@ -175,6 +247,175 @@ TEST_F(ThreadTest, StartOneThreadTwice_ExpectTheSameThreadStartedTwice) {
   EXPECT_TRUE(delegate_->check_value());
 }
 
+TEST(ThreadTest2, StartAndJoinForce_Success) {
```

Comment:
@pvvasilev Usually the test name should refelect what it does. So I'm agree with @AGritsevich that `ThreadTest2` tells me nothing